### PR TITLE
Enable set the icons map

### PR DESCRIPTION
**Pull request by mikicho**
I want to override the default icons to support custom blocks. I don't know if this is the best way to achieve this. Please let me know what you think.

### CODE REVIEW

**Comment by m-Ryan** [comment]
https://github.com/zalify/easy-email-editor/issues/153

Our recommended way is to override the CSS corresponding to the class name.

**Comment by mikicho** [comment]
I saw this issue. Why don't we want to expose the icon set map?

File: packages/easy-email-extensions/src/utils/getIconNameByBlockType.ts [hunk 1]
```diff
@@ -38,3 +38,7 @@ const iconsMap = {
 export function getIconNameByBlockType(type: string) {
   return get(iconsMap, type) || 'icon-number';
 }
+
+export function setIconsMap(map: Record<string, string>) {
+  iconsMap = {...map};
+}
```
Comment (m-Ryan) [inline packages/easy-email-extensions/src/utils/getIconNameByBlockType.ts:44]: export function setIconsMap(map: Record<string, string>) {
  iconsMap = {...iconsMap, ...map};
}

**Comment by m-Ryan** [comment]
> I saw this issue. Why don't we want to expose the icon set map?

Doing it this way is also fine. For small PRs, I will bundle a few features together before releasing them, rather than releasing immediately.

**Comment by mikicho** [comment]
> Doing it this way is also fine. For small PRs, I will bundle a few features together before releasing them rather than releasing them immediately.

Sure! no rush, for now I'm using the `css` workaround

**Review by m-Ryan — APPROVED** [review]

**Pull request merged**